src/Xenial.Tasty/FeatureDetection/Win/ProcessExtensions.Toolhelp32.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+
+namespace Xenial.Delicious.FeatureDetection.Win
+{
+    internal static class Toolhelp32
+    {
+        internal const uint Inherit = 0x80000000;
+        internal const uint SnapModule32 = 0x00000010;
+        internal const uint SnapAll = SnapHeapList | SnapModule | SnapProcess | SnapThread;
+        internal const uint SnapHeapList = 0x00000001;
+        internal const uint SnapProcess = 0x00000002;
+        internal const uint SnapThread = 0x00000004;
+        internal const uint SnapModule = 0x00000008;
+
+        [DllImport("kernel32.dll")]
+        static extern bool CloseHandle(IntPtr handle);
+        [DllImport("kernel32.dll")]
+        static extern IntPtr CreateToolhelp32Snapshot(uint flags, int processId);
```

Comment:
return type should be a SafeHandleMinusOneIsInvalid

Reply:
Where does this live?

Reply:
See [https://docs.microsoft.com/en-us/dotnet/api/microsoft.win32.safehandles?view=netstandard-2.0](https://docs.microsoft.com/en-us/dotnet/api/microsoft.win32.safehandles?view=netstandard-2.0)